qa/tasks/cephfs/mount.py

```diff
@@ -780,7 +780,7 @@ def getfattr(self, path, attr):
         try:
             p.wait()
         except CommandFailedError as e:
-            if e.exitstatus == 1 and "No such attribute" in p.stderr.getvalue():
+            if e.exitstatus == 1 and b"No such attribute" in p.stderr.getvalue():
```

Comment:
This method:
```
    def getfattr(self, path, attr):
        """
        Wrap getfattr: return the values of a named xattr on one file, or
        None if the attribute is not found.
        :return: a string
```
But I see this:
```
        return p.stdout.getvalue()
```
Have we havde decode stdout here since we're sure the stdout is io.Bytes?
Also I can see in the code:
```
return None
```
Shouldn't we change description like:
```
        :return: a string, or None if attribute missing 
```

Reply:
i'd say this is not in the scope of this change, as the commit message does not claim to fix this issue. do you mind sending a PR to address this?